### ISSUE TITLE
Unparsed group "POB000"

### ISSUE THREAD

**Issue by christophermclement** [python-metar/python-metar#141]
From 1981-08-31, CYEG:
Unparsed groups in body 'POB000' while processing 'CYEG 310100Z AUTO 00000KT 3SM POB000 BKN034 10/05 A2993 RMK SLP154 T01000050 IEM_DS3505'

POB, insofar as I can tell, means "Persons on Board," but I can't understand why that would be in a METAR?

For reference, among the tiny sliver of METARs I've looked at, the issue is also seen on:
1981-11-15 CYEG
1989-08-31
1990-11-11
1993-11-07
1995-11-05
1993-08-06 CYHM
1994-07-14
1994-07-28
1995-07-25
1995-07-28
1995-08-20
1980-06-18 CYOW
1980-07-08
1980-07-29
1980-08-05
1981-09-04
1980-09-03
1983-06-16
1983-10-29
1984-06-06
1986-06-27
1986-07-24
1986-08-15
1986-10-13
1986-11-01
1987-06-26
1988-08-02
1988-10-22
1981-08-09 CYQR
1982-07-30
1983-07-29
1990-10-28
1991-10-27
1994-06-29
1980-06-25 CYUL
1980-08-26
1980-09-21
1982-08-27
1983-09-17
1986-08-07
1984-06-21 CYWG
1986-11-02
1988-11-13
1989-08-02
1990-08-08
1993-11-21
1994-09-23
1982-06-29
1985-10-25 CYYC
1988-09-05
1992-11-08
1982-09-26 CYYZ
1983-08-26
1983-10-23
1983-11-20
1984-07-22
1984-10-14
1984-10-27
1984-11-11
1986-08-08
1987-10-16
1988-07-28
1988-11-20
1988-06-26
1995-07-15 KBHM

**Comment by jhaiduce** [comment]
Maybe POB is a typo for PROB?

**Comment by christophermclement** [comment]
Hundreds of times though? The list above is only checking about 50 METARs per year per site.

I searched all of CYOW by year for "POB" and found:
1970 and earlier-0
1971-1976-No METARs available
1977-579
1978-780
1979-1196
1980-910
1981-893
1982-992
1983-1076
1984-1235
1985-951
1986-1090
1987-980
1988-718
1989-11, 8 in one block
1990-6, 5 in one block
1991-0
1992-2
1993-11
1994-0
1995-3
1996 and beyond-0

I can only assume it meant something in that period, and then for whatever reason they stopped including that information except occasionally, finally dropping it completely.

**Comment by jhaiduce** [comment]
> Hundreds of times though? The list above is only checking about 50 METARs per year per site.

Yeah, that's weird. And PROB is code for "probability," normally found in TAFs not METARs.

**Comment by christophermclement** [comment]
Very interesting. A sampling of some "POB000" METARs from CYOW, January 1986

CYOW 070838Z AUTO 07003KT 2 1/2SM POB000 OVC039 RMK IEM_DS3505
CYOW 071100Z AUTO 00000KT 1 1/4SM POB000 OVC010 M04/M04 A2950 RMK SLP997 T10401040 IEM_DS3505
CYOW 101100Z AUTO 07009KT 2SM POB000 OVC002 M03/M03 A2995 RMK SLP149 T10301030 IEM_DS3505
CYOW 181300Z AUTO 09016KT 4SM POB000 OVC079 M14/M18 A3000 RMK SLP173 T11401180 IEM_DS3505
CYOW 070108Z AUTO 10009KT 3SM POB000 BKN007 OVC016 RMK IEM_DS3505

A consistent feature (also seen in others that I didn't include here) is overcast clouds at low altitudes, so instead of superseding the other cloud groups it appears to be concurrent with them?

**Comment by christophermclement** [comment]
And yes, all have poor visibility and similar temp/dewpoint
